mempool/src/feerate.rs

```diff
@@ -0,0 +1,54 @@
+use common::primitives::amount::Amount;
+
+lazy_static::lazy_static! {
+    pub(crate) static ref INCREMENTAL_RELAY_FEE_RATE: FeeRate = FeeRate::new(1000);
+    pub(crate) static ref INCREMENTAL_RELAY_THRESHOLD: FeeRate = FeeRate::new(500);
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct FeeRate {
+    tokens_per_kb: u128,
```

Comment:
If "tokens" here equivalent satoshis, then please call it "atoms".

Reply:
Done.